lib/Controller/PersonController.php

```diff
@@ -63,6 +63,8 @@ public function index() {
 		$model = intval($this->config->getAppValue('facerecognition', 'model', AddDefaultFaceModel::DEFAULT_FACE_MODEL_ID));
 
 		$resp = array();
+		$resp['enabled'] = $this->config->getUserValue($this->userId, 'facerecognition', 'enabled', false);
```

Comment:
if not enabled, should we send persons to client at all? Why just not "fake" it and send empty array?

Reply:
Oh .. You're right... :+1: